hera_pspec/pspecbeam.py

```diff
@@ -1,24 +1,167 @@
-import numpy as np, pyuvdata, os, conversions
+import numpy as np
+import pyuvdata
+import os
+from hera_pspec import conversions
 from scipy import integrate
 from scipy.interpolate import interp1d
+import aipy
+
+
+def _compute_pspec_scalar(cosmo, beam_freqs, omega_ratio, pspec_freqs, num_steps=20000,
```

Comment:
Is there a reason the default `numsteps` parameter here does not match the defaults elsewhere in the file?

Reply:
nope, i guess just a typo..

Reply:
fixed